JavaScript/Twitter_Unfollowers/README.md

```diff
@@ -0,0 +1,24 @@
+# Twitter_Unfollowers
+**Twitter_Unfollowers** is a script to track people that recently unfollowed you on Twitter.
+- It uses Twitter API to fetch followers of any user *(provided as input)*.
+- Then stores it into a json file. 
+- When compared again it gives name and number of people that unfollowed.
+- *(Additionally) :* After multiple queries it provides search/query history in tabular form.
+## Setup instructions
+- Get your [Twitter developer account](https://developer.twitter.com/en) and follow the steps there to generate your **bearer key**
+- create a file named `apiTokens.js` in the Twitter_Unfollowers folder and enter the following : ![File content](https://i.imgur.com/4UlveGE.png)
```

Comment:
Would it be possible to have the file inside the directory itself so that we can copy and paste our key over there :)